tests/dummy/app/wizard-example/template.hbs

```diff
@@ -0,0 +1,48 @@
+{{#usage-example title='Wizard example'}}
```

Comment:
Great idea, adding a full-blown "wizard" example. It's been something I wanted to do for a while -- thank you!